guide/english/angular/routing/index.md

```diff
@@ -462,6 +462,8 @@ Routing techniques apply regardless of a `Route` object’s level of nesting. Th
 
 Web applications often consist of public and private data. Both types of data tend to have their own pages with *guarded* routes. These routes allow/restrict access depending on the user’s privileges. Unauthorized users may interact with a guarded route. The route should block the user if he or she attempts to access its routed content.
 
+One example of why you would have an guarded route is to prevent someone not meant to access a route accessing a particular route by just knowing the endpoint.
```

Comment:
Thank you for your Pull Request.  I think this sentence could be reworded to something more like below, to make it more concise:
> One example of why you would have an guarded route, is to restrict unauthorized users who just happen to know the endpoint.